Cesium3DTilesContent/include/Cesium3DTilesContent/LegacyUtilities.h

```diff
@@ -0,0 +1,133 @@
+#pragma once
+
+#include <Cesium3DTilesContent/GltfConverters.h>
+#include <CesiumAsync/IAssetAccessor.h>
+#include <CesiumAsync/IAssetRequest.h>
+#include <CesiumGltf/AccessorView.h>
+#include <CesiumGltf/PropertyTransformations.h>
+#include <CesiumUtility/ErrorList.h>
+
+#include <glm/fwd.hpp>
+#include <rapidjson/document.h>
+
+#include <cstdint>
+#include <optional>
+#include <vector>
+
+namespace CesiumGltf {
+class Model;
+class Buffer;
+} // namespace CesiumGltf
+
+namespace Cesium3DTilesContent {
+
+struct ByteResult {
+  std::vector<std::byte> bytes;
+  CesiumUtility::ErrorList errorList;
+};
+
+CesiumAsync::Future<ByteResult>
+get(const ConverterSubprocessor& subprocessor, const std::string& relativeUrl);
+
+namespace LegacyUtilities {
```

Comment:
If I may nitpick... the name `LegacyUtilities` has the possibility of sounding like the "utilities" themselves are legacy, instead of being utility for legacy content.

But I also think it's okay to forgo the "legacy" entirely and just say

```suggestion
namespace GltfConverterUtility {
```

or something similar.

Reply:
I was thinking of Tiles10Utility.